build/dsc/templates/dsc_type.rb.erb

```diff
@@ -4,6 +4,7 @@ Puppet::Type.newtype(:dsc_<%= resource.friendlyname.downcase %>) do
   require Pathname.new(__FILE__).dirname + '../../' + 'puppet/type/base_dsc'
 
   provide :powershell, :parent => Puppet::Type.type(:base_dsc).provider(:powershell) do
+    confine :powershell_major_version => '5'
```

Comment:
Should we also do a minor / patch version here?  We know that we want at least the production preview for the LCM changes.